scripts/post-pip-compile.sh

```diff
@@ -17,7 +17,7 @@ function clean_file {
     # Workaround for https://github.com/jazzband/pip-tools/issues/204 -
     # change absolute paths for local editable packages back to relative ones
     FILE_CONTENT=$(<${FILE_PATH})
-    FILE_URL_REGEX="-e (file:///[^"$'\n'"]*)/common/lib/symmath"
+    FILE_URL_REGEX="-e (file:///[^"$'\n'"]*)/common/lib/xmodule"
```

Comment:
@Carlos-Muniz I think the problem is that `py38.txt` (the file that ended up with the extra `-e file://...` in it) doesn't have include `common/lib/xmodule`, so it didn't match this regex and therefore never got its line for `common/lib/sandbox-packages` fixed.

What if we changed the regex to match any `common/lib/<packagename>` path? Something like:
```
FILE_URL_REGEX="-e (file:///[^"$'\n'"]*)/common/lib/\w+"
```

Reply:
I think you're right.
And I think https://github.com/openedx/edx-platform/pull/29918 proves it

Reply:
Nice!